tests/series/arithmetic/test_mul.py

```diff
@@ -2,17 +2,23 @@
 
 import numpy as np
 import pandas as pd
+import pytest
 from typing_extensions import assert_type
 
 from tests import (
     TYPE_CHECKING_INVALID_USAGE,
     check,
 )
 
-left_i = pd.DataFrame({"a": [1, 2, 3]})["a"]  # left operand
 
+@pytest.fixture
+def left_i() -> "pd.Series[Any]":
+    """left operand"""
+    lo = pd.DataFrame({"a": [1, 2, 3]})["a"]
+    return check(assert_type(lo, pd.Series), pd.Series)
 
-def test_mul_py_scalar() -> None:
+
+def test_mul_py_scalar(left_i: "pd.Series[Any]") -> None:
```

Comment:
Same comment as above with `test/index/arithmetic/test_mul.py` - remove the `Any`